libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -173,16 +203,29 @@ DNNModel *ff_dnn_load_model_ov(const char *model_filename, const char *options)
     if (!ov_model)
         goto err;
     ov_model->ctx.class = &dnn_openvino_class;
+    ctx = &ov_model->ctx;
 
     status = ie_core_create("", &ov_model->core);
     if (status != OK)
         goto err;
 
+    status = ie_core_get_available_devices(ov_model->core, a_dev);
+    if (status != OK)
+        goto err;
+    if (options && parse_options(&ov_model->ctx, options, a_dev) == DNN_ERROR) {
+        av_log(ctx, AV_LOG_ERROR, "Failed to parse options \"%s\"\n", options);
+        goto err;
+    }
+    if (!ov_model->ctx.device_type) {
+        av_log(ctx, AV_LOG_WARNING, "No proper device specified, will run on CPU defaultly\n");
+        ov_model->ctx.device_type = "CPU";
+    }
+
     status = ie_core_read_network(ov_model->core, model_filename, NULL, &ov_model->network);
     if (status != OK)
         goto err;
 
-    status = ie_core_load_network(ov_model->core, ov_model->network, "CPU", &config, &ov_model->exe_network);
+    status = ie_core_load_network(ov_model->core, ov_model->network, ov_model->ctx.device_type, &config, &ov_model->exe_network);
     if (status != OK)
         goto err;
```

Comment:
list all the supported devices if status is not OK

Reply:
OK, we should list all available device rather than use CPU defaultly, right?
Will modify in next patch.

Reply:
no, the default is "CPU", I mean the code like:

if (status != OK) {
    output message to list all the supported devices
    goto err;
}